compiler/src/symboltable.rs

```diff
@@ -97,10 +97,13 @@ pub struct Symbol {
     pub name: String,
     // pub table: SymbolTableRef,
     pub scope: SymbolScope,
+    // TODO(lynskylate@gmail.com): Use bitflags replace
```

Comment:
I don't want to go with TODO ownership in the code. How about creating and assigning an issue if you want to mark some issue for you?

Reply:
Done.